cloud/amazon/ec2_elb.py

```diff
@@ -98,69 +98,87 @@
     with_items: ec2_elbs
 """
 
+RETURN = '''
+ansible_facts:
+    description: a dict with the key "ec2_elbs" with as value the list names of lb managed.
+    returned: success
+    type: dict
+    sample: "ec2_elbs": ["lb-name1", "lb-name2", "lb-name3"]}
+elb_changed:
+    description: a dict with lb name has key and for each name the list of instance_id updated.
+    returned: success
+    type: dict
+    sample: { "lb-name1": ["i-12345678","i-34567890"], "lb-name2": [], "lb-name3": ["i-12345678"] }
+wait_attempt:
+    description: a dict with lb name has key and for each name the number of attempt to have expected state for all these instances. Make sense only with wait true. 
+    returned: success
+    type: dict
+    sample: { "lb-name1": 4, "lb-name2": 0, "lb-name3": 1 }
+'''
+
 import time
+import sys
+import os
+import copy
 
 try:
     import boto
     import boto.ec2
-    import boto.ec2.autoscale
     import boto.ec2.elb
     from boto.regioninfo import RegionInfo
-    HAS_BOTO = True
 except ImportError:
-    HAS_BOTO = False
-
+    print "failed=True msg='boto required for this module'"
+    sys.exit(1)
 
 class ElbManager:
     """Handles EC2 instance ELB registration and de-registration"""
 
-    def __init__(self, module, instance_id=None, ec2_elbs=None,
+    def __init__(self, module, instance_ids=[], ec2_elbs=None,
```

Comment:
it is not a good idea to initialize list in the method definition, keep it None and check for it inside the method